README.md

```diff
@@ -150,6 +150,48 @@ should be configured to neither warn about nor overwrite _Black_'s changes.
 Actual details on _Black_ compatible configurations for various tools can be found in
 [compatible_configs](https://github.com/psf/black/blob/master/docs/compatible_configs.md).
 
+### Migrating your code style without ruining git blame
+
+A long-standing counterargument against moving to automated code formatters like _Black_
+is that the migration will clutter up the output of `git blame`. This used to be a valid
+argument, but not anymore. Since `git` version 2.23, git natively supports
```

Comment:
```suggestion
is that the migration will clutter up the output of `git blame`. This was a valid
argument, but since `git` version 2.23, git natively supports
```